include/boost/geometry/extensions/gis/projections/impl/pj_init.hpp

```diff
@@ -204,10 +206,23 @@ inline parameters pj_init(R const& arguments, bool use_defaults = true)
 
     if (! s.empty())
     {
-        // TODO: IMPLEMENT SPLIT
-        pin.to_meter = atof(s.c_str());
+        std::size_t const pos = s.find('/');
+        if (pos == std::string::npos)
+        {
+            pin.to_meter = lexical_cast<double>(s.c_str());
+        }
+        else
+        {
+            double const numerator = lexical_cast<double>(s.substr(0, divisor)), 
```

Comment:
Ahhh, haha, I *thought* I compiled it but it is the recurring I problem I have with working on Boost that my development copy is not my 'working' copy. I don't like bjam all that much so I never learnt to use the test suites but I probably should.